pandas/io/formats/style_render.py

```diff
@@ -798,6 +798,7 @@ def format(
         decimal: str = ".",
         thousands: str | None = None,
         escape: str | None = None,
+        hyperlinks: str | None = None,
```

Comment:
is there a reason we prefer this to `render_links`?

Reply:
My arguments: `hyperlink` is a bit more specific whereas `link` can be a general term in certain contexts. And, technically, everything is rendered so why not add `render` to every kwarg. And rendering is somewhat of an esoteric concept outside of graphics processing.

Reply:
ok fair